### ISSUE TITLE
number of repochs

### ISSUE THREAD

**Issue by mahilaMoghadami** [akhilpm/DroneDetectron2#16]
hello
please tell me number of iteration you set for training to achieve AP 58.3 on VisDrone dataset?
I trained on default number of epochs and I get AP = 32

this is my train command and config file I used:
python train_net.py --num-gpus 1 --config-file configs/RCNN-FPN-CROP.yaml OUTPUT_DIR outputs_FPN_CROP_VisDrone

and this is my last part of training log:
![image](https://github.com/akhilpm/DroneDetectron2/assets/52579373/b13f9c64-e5e3-4983-8bc6-569b9e1dffd6)
thank you

**Comment by akhilpm** [comment]
You should be getting AP above 32 within 90k iterations (as shown in the config file). My base FPN (Base-RCNN-FPN.yaml) model is also giving much higher than your results shown here. Check where your Base FPN stands first to make sure that the setup is correct before using the crops. Also, one reason might be you are not using ImageNet pre-trained weights for the ResNet, instead the random weight initialization.

Note that I also used a higher LR of 0.01 and clipped the gradients.

**Comment by mahilaMoghadami** [comment]
Hello
I trained model with 20K iteration and I get AP0.5 37.14.
how you analyse it?
could you please share ImageNet pre-trained weights for the ResNet ?
and whats you idea to improve it and get results same as your paper?
